OverlayPlugin.Core/NetworkProcessors/LineActorSetPos.cs

```diff
@@ -0,0 +1,131 @@
+﻿using System;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using RainbowMage.OverlayPlugin.MemoryProcessors;
+
+namespace RainbowMage.OverlayPlugin.NetworkProcessors
+{
+    [StructLayout(LayoutKind.Explicit, Size = structSize, Pack = 1)]
+    internal unsafe struct ActorSetPos_v655
+    {
+        // 6.5.5 packet data (minus header):
+        // 6AD3 0F  02  00000000 233E3BC1 00000000 D06AF840 00000000
+        // AAAA BB  CC  DDDDDDDD EEEEEEEE FFFFFFFF GGGGGGGG HHHHHHHH
+        // 0x0  0x2 0x3 0x4      0x8      0xC      0x10     0x14
+        // Rot  unk unk unk      X        Y        Z        unk
+
+        // Have never seen data in 0x4 or 0x14, probably just padding?
+
+        public const int structSize = 24;
+        [FieldOffset(0x0)]
+        public ushort rotation;
```

Comment:
Is there a difference between `rotation` and `heading`?  I see `heading` in the `ActorMove` struct and that's how I've seen it referred to, so not sure if `rotation` has a different meaning (if so, an explanatory comment would be great).

Reply:
Traditionally:
- `rotation` would be a value from `-π` to `+π`
- `heading` would be a 2D or 3D vector

But the language is mixed elsewhere in the project.

- `FFXIV_ACT_Plugin` uses `rotation`.
- `FFXIVClientStructs` uses `rotation`.
- `OverlayPlugin`'s combatant-related classes and structures use `heading`.
- `OverlayPlugin`'s `LineActorCastExtra` and `LineAbilityExtra` use `rotation`.

I think it would make sense for everything new going forward to use the correct terminology of `rotation`.